package/yast2-services-manager.spec

```diff
@@ -24,7 +24,7 @@
 ######################################################################
 
 Name:           yast2-services-manager
-Version:        3.1.43
+Version:        3.1.44
```

Comment:
Please, use `3.2.0` for the `master` branch. `master` -> 3.2.x, `SLE-12-SP2` -> 3.1.44 and `SLE-12-SP2-CASP` -> 3.1.44.1 (a fourth number).